natlas-server/run-server.sh

```diff
@@ -8,17 +8,18 @@ source .env
 source venv/bin/activate
 DEPLOY_ENV=${FLASK_ENV:-production}
 
-if [[ ! -d logs ]]; then
+if [[ ! d logs ]]; then
 	mkdir logs
 fi
 
-if [ $DEPLOY_ENV == "development" ]
+if [ "$DEPLOY_ENV" == "development" ]
 then
-	echo "`date` : Development" >> logs/start.log
-	echo "`date` : Development"
+	echo "$(date) : Development" >> start.log
```

Comment:
We're almost there. Tack a `logs/` in front of start.log on this file and the one one L21 and then I think it's ready to go.